tests/rt.sh

```diff
@@ -425,7 +425,7 @@ fi
 if [[ $MACHINE_ID = cheyenne.* ]]; then
   RTPWD=${RTPWD:-$DISKNM/develop-20200210/${COMPILER^^}}
 else
-  RTPWD=${RTPWD:-$DISKNM/FV3-MOM6-CICE5/develop-20200718}
+  RTPWD=${RTPWD:-$DISKNM/FV3-MOM6-CICE5/develop-20200530}
```

Comment:
why isn't this the same date as the develop branch?

Reply:
I think @MinsukJi-NOAA said the 35d files were being held there, but i don't think thats being used. So we can keep the current date/time.

Reply:
Yeah, to run 35d 'warm' start, we need to point to 20200530 for mediator data. But it should be changed back to the latest when committing the codes.

Reply:
OK i'll get that back to the old one!